### PR TITLE
Bugs

### PR DESCRIPTION
**Pull request by prwelber**
Mo Fixes Less Problems

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js [hunk 1]
```diff
@@ -105,44 +105,60 @@ class AcceptTerms extends Component {
             <FieldsContainer>
               <FieldContainer>
                 <Text size='14px' style={spacing('mb-10')}>
-                  <FormattedMessage id='sfoxexchangedata.create.createaccount.partner.verifiedemail' defaultMessage='Verified Email Address' />
+                  {
+                    emailVerified
+                      ? <FormattedMessage id='sfoxexchangedata.create.createaccount.partner.verifiedemail' defaultMessage='Verified Email Address' />
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js]: How can you get to this step if your email is not verified?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Login/template.js [hunk 2]
```diff
@@ -120,7 +120,11 @@ const Login = (props) => {
         </FormGroup>
         <FormGroup>
           <LoginButton type='submit' nature='primary' fullwidth uppercase disabled={submitting || invalid}>
-            <FormattedMessage id='scenes.login.submit' defaultMessage='Log in' />
+            {
+              busy
+                ? <HeartbeatLoader height='20px' width='20px' color='white' />
+                : <FormattedMessage id='scenes.login.submit' defaultMessage='Log in' />
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Login/template.js]: @schnogz just FYI looks like this was done, know you were thinking of working on this.

File: packages/blockchain-wallet-v4-frontend/src/services/ValidationHelper/index.js [hunk 3]
```diff
@@ -114,6 +114,8 @@ const isUsZipcode = val => {
   return val
 }
 
+const formatPhone = (val, prevVal) => val.replace(/[^\d]/g, '')
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/services/ValidationHelper/index.js]: prevVal not used

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBox/index.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ const SelectBox = (props) => {
   return (
     <Container>
       <SelectInput {...input} {...meta} {...rest} errorState={errorState} />
-      {(touched || !pristine) && error && <Error>{error}</Error>}
+      {(touched || !pristine) && error && <Error errorBottom={props.errorBottom}>{error}</Error>}
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBox/index.js]: Thanks for adding this.

File: packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js (hunk 1 above)
Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js]: With unverified email, -->  buy/sell tab -->  modal opens to verifyEmail --> click cancel. 
On cancel, we can either close the modal or return user to AcceptTerms. I figured keeping them in the flow is better than closing modal (which already has a close mechanism anyway).

File: packages/blockchain-wallet-v4-frontend/src/services/ValidationHelper/index.js (hunk 3 above)
Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/services/ValidationHelper/index.js]: good spot!

File: packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js (hunk 1 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js]: Shouldn't it take you back to change email step?

**Pull request merged**